app/views/users/_form.html.erb

```diff
@@ -13,12 +13,12 @@
 
   <div class="container">
     <%= form.label :name %>
-    <%= form.text_area :name %>
+    <%= form.text_field :name %>
   </div>
 
   <div id="non-editable" class="field">
     <%= form.label :email %>
-    <%= form.text_area :email%>
+    <%= form.email_field :email%>
   </div>
 
```

Comment:
@HrisikeshSinghaChowdhury In this form also you need to add options to upload image so that for existing users they can upload or edit their image

Reply:
That was the point....i am facing error in edit....new pic is uploaded but when being displayed....it is showing the first pic......so in that case should i need to write a query to get the last pic from db?